<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Renombrado de imágenes y etiquetas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="otrosDatos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Otros Datos</string>
    </property>
    <widget class="QLabel" name="prefixLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prefijo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prefixName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Prefijo</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxDigitsLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Máx. dígitos en nombre</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="DigitsSpin">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Renombrar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Renombrar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GrupoDeFicheros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Selección de rutas</string>
    </property>
    <widget class="QLineEdit" name="ImagesDirTextLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Imágenes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ImagesDirSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Imágenes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LabesDirTextLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Etiquetas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LabelsDirSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Etiquetas</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
